From a838782758570cc7aaa0afbbc079374a1a1b8f8a Mon Sep 17 00:00:00 2001
From: Guilherme Brondani Torri <guitorri@gmail.com>
Date: Thu, 2 Nov 2017 21:21:15 +0100
Subject: [PATCH 1/3] DYLD_LIBRARY_PATH behave as LD_LIBRARY_PATH on OSX

It is needed otherwise MODELGEN won't be able to find libgnucap
at build time.
---
 apps/Make1 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/apps/Make1 b/apps/Make1
index 95bce14..acbbf55 100644
--- a/apps/Make1
+++ b/apps/Make1
@@ -123,9 +123,9 @@ LIBPATH = `pwd | sed 's/apps/lib/g'`
 MODELGEN = `pwd | sed 's/apps/modelgen/g'`/gnucap-modelgen
 #------------------------------------------------------------------------
 %.h : %.model
-	(export LD_LIBRARY_PATH=$(LIBPATH); $(MODELGEN) -h $<)
+	(export DYLD_LIBRARY_PATH=$(LIBPATH); $(MODELGEN) -h $<)
 #------------------------------------------------------------------------
 %.cc : %.model
-	(export LD_LIBRARY_PATH=$(LIBPATH); $(MODELGEN) -cc $<)
+	(export DYLD_LIBRARY_PATH=$(LIBPATH); $(MODELGEN) -cc $<)
 #-----------------------------------------------------------------------------
 #-----------------------------------------------------------------------------
-- 
2.11.1

